Week1/node.js

```diff
@@ -0,0 +1,184 @@
+  "INSERT INTO country (name, continent,surface_area, population) VALUES ('Tanzania', 'Africa', 883749, 57310019)",
+  "INSERT INTO country (name, continent,surface_area, population) VALUES ('Netherlands','Europe', 800, 17035938)",
+  "INSERT INTO country (name, continent,surface_area, population) VALUES ('Afghanistan', 'Asia',  652090, 35530081)",
+  "INSERT INTO country (name, continent,surface_area, population) VALUES ('Romania','Europe', 238391, 19679306)",
+  "INSERT INTO country (name, continent,surface_area, population) VALUES ('Switzerland','Europe', 41284, 8476005)",
+  "INSERT INTO country (name, continent,surface_area, population) VALUES ('Sweden', 'Europe', 449964, 9910701)",
+];
+for (let i in insert_countries) {
+  database.query(insert_countries[i], error => {
+    if (error) {
+      throw error;
+    }
+    console.log([i] + 'success');
+  });
+}
+
+var insert_cities = [
+  "INSERT INTO city (id, name, country_code, population) VALUES (1, 'Amsterdam', 'NLD', 821752)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (2, 'Rotterdam','NLD', 623652)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (3, 'Kabul','KAB', 4600000)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (4, 'Bucharest','BUC', 1836000)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (5, 'Stockholm','STO',  965232)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (6, 'Damascus', 'DAM',1711000)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (7, 'Dodoma', 'DOD',2084000)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (8, 'Bern', 'BER', 133115)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (9,'Arnhem','NLD',138020)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (10,'Breda','NLD',160398)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (11,'Huambo','HUA',163100)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (12,'Lobito','LOB',130000)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (13,'Benguela','BEN',128300)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (14,'Zaanstad', 'ZAA',135621)",
+];
+
+for (let i in insert_cities) {
+  // console.log('Going to run ', insert_cities[i]);
+  database.query(insert_cities[i], error => {
+    if (error) {
+      throw error;
+    }
+    console.log([i] + 'success');
+  });
+}
+
+// What are the names of the countries with population greater than 8 million
+database.query(`SELECT * FROM country WHERE Population > 8000000`, (error, result) => {
+  if (error) throw error;
+
+  console.log(`\n Countries with population more than 8 million:`);
+  result.forEach(country => {
+    console.log(`${country.Name} , ${country.Population}`);
+  });
+});
+
+// Names of the countries that have “land” in their names ?
+database.query(`SELECT * FROM country WHERE Name LIKE '%land%' `, (error, result) => {
```

Comment:
Using `select *` is not recommended in real systems. Be specific with the result and only return the columns that you need, in this case the name.

Reply:
Thanks , i make it name